src/launch/policy.c

```diff
@@ -305,8 +305,10 @@ static int policy_import_connect(Policy *policy, ConfigNode *cnode) {
         }
 
         if (cnode->parent->policy.context == CONFIG_POLICY_USER ||
-            cnode->parent->policy.context == CONFIG_POLICY_GROUP) {
-                fprintf(stderr, "Connection policy not allowed in user/group context in %s +%lu\n",
+            cnode->parent->policy.context == CONFIG_POLICY_GROUP ||
+            cnode->parent->policy.context == CONFIG_POLICY_NO_CONSOLE ||
+            cnode->parent->policy.context == CONFIG_POLICY_AT_CONSOLE) {
```

Comment:
I don't think that's correct. Connect-policies are allowed in at_console. Grep for `BUS_POLICY_RULE_IS_PER_CLIENT` in dbus-daemon.

Reply:
Ah, indeed. Good catch. Fixed.